src/framework/components/collision/system.js

```diff
@@ -330,12 +332,15 @@ class CollisionMeshSystemImpl extends CollisionSystemImpl {
     // special handling
     beforeInitialize(component, data) {}
 
-    createAmmoMesh(mesh, node, shape) {
-        let triMesh;
+    createAmmoMesh(mesh, node, shape, checkDuplicates) {
```

Comment:
```suggestion
    createAmmoMesh(mesh, node, shape, checkDuplicates = true) {
```
I would default this to false to avoid possibly passing `undefined` to the API if the data does not contain it.

Reply:
I will default it to true. I am assuming your comment has a typo, suggesting it to be a `false`.